diff --git a/PKGBUILD b/PKGBUILD
index de8530e..3dff7c2 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -178,11 +178,21 @@ build() {
   ulimit -n 4096
 
   # Do 3-tier PGO
-  echo "Building instrumented browser..."
+##  echo "Building instrumented browser..."
+#XXX: temp disabled to avoid re-configure running. actually allowing this doesn't re-configure it
   cat >.mozconfig ../mozconfig - <<END
-ac_add_options --enable-profile-generate=cross
+#ac_add_options --enable-profile-generate=cross
+#ac_add_options --enable-debug         # Changed: Debug symbols ON
+ac_add_options --enable-debug-symbols
+#ac_add_options --disable-optimize     # Changed: No optimization for debug
+ac_add_options --enable-optimize
 END
   ./mach build --priority normal
+  #./mach build --priority normal --debug netwerk/protocol/http netwerk/ipc ipc/glue && mach run --debugger=gdb
+  #./mach run --debugger=gdb -P
+  #./mach run -P
+  echo "ok build once"
+  exit 0
 
   echo "Profiling instrumented browser..."
   ./mach package
